### ISSUE TITLE
Add ability to define the dir to be mount to the Docker

### ISSUE THREAD

**Issue by agentmilindu** [leopardslab/Dunner#4]
Users should be able to define the sub-directory of the source directory which should be mounted as a volume to the Docker as `dir`  

```
deploy:
- name: 'node'
  dir: 'frontend'
  command: ['sonar', 'scan']
- name: 'mvn'
  dir: 'backend'
  command: ['mvn', 'build']
```

**Comment by vaibhavk** [comment]
Can I work on this issue?

**Comment by agentmilindu** [comment]
yes, please go ahead @vaibhavk

**Comment by ayushjn20** [comment]
This issue can be closed with reference to #45.

**Comment by ayushjn20** [comment]
This issue can be closed with reference to #48.

**Comment by agentmilindu** [comment]
Fixed by #48
